### PR TITLE
Up5 readme

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: cdap-ui/README.rst [hunk 1]
```diff
@@ -1,4 +1,108 @@
...
+
+- ``/opt/cdap`` on RPM (manual or Apache Ambari) installations; or
+- the *CDAP* sub-directory of the *Parcel Directory* on Cloudera Manager installations (e.g. ``/opt/cloudera/parcels/CDAP``)
+
+::
+
+  $ cd <CDAP_HOME>
+  $ /etc/init.d/cdap-ui stop
+  $ cdap apply-pack /path/to/download/cdap-ui-pack-4.2.0_p5.zip
+  $ /etc/init.d/cdap-ui start
```
Comment (elfenheart) [inline cdap-ui/README.rst:42]: Probably should add a disclaimer that Node version > 4.5.0 needs to be installed

**Comment by joefarned** [comment]
LGTM

File: cdap-ui/README.rst [hunk 2]
```diff
@@ -1,4 +1,108 @@
...
+
+Release Notes
+=============
+
+New Features
+------------
+
+Actions
+CDAP-9107
+Allow users to undo/redo node deletion
```
Comment (tonybach) [inline cdap-ui/README.rst]: Seems like you're missing a `*`, which is screwing up the formatting

File: cdap-ui/README.rst [hunk 3]
```diff
@@ -1,4 +1,108 @@
...
+* `CDAP-11992 <https://issues.cask.co/browse/CDAP-11992>`__ - Handled errors correctly in the Data Prep UI
+* `CDAP-11827 <https://issues.cask.co/browse/CDAP-11886>`__ - Added scrolling to the data prep dropdown menu
+* `CDAP-11627 <https://issues.cask.co/browse/CDAP-11627>`__ - Fixed the extract fields dropdown in data prep to scroll when it doesn't fit the screen size
+* `CDAP-11438 <https://issues.cask.co/browse/CDAP-11438>`__ - Selected the entire column name when the user wants to rename it
+* `CDAP-8419 <https://issues.cask.co/browse/CDAP-8419>`__ - Fixed the height calculation for the add new entity modal
+
+Known Issues
+------------
+* `CDAP-11943 <https://issues.cask.co/browse/CDAP-11943>`__ - Due to a known issue, even after upgrading the Data Preparation version that ships with CDAP 4.2,
+after restarting CDAP, CDAP may use the older version of Data Preparation. As a workaround, click the Upgrade button that shows up on the main page of Data Preparation
```
Comment (tonybach) [inline cdap-ui/README.rst:97]: Don't start a new line here? The formatting here looks weird right now

File: cdap-ui/README.rst (hunk 2 above)
Comment (bdmogal) [reply cdap-ui/README.rst]: fixed

File: cdap-ui/README.rst (hunk 1 above)
Comment (bdmogal) [reply cdap-ui/README.rst:42]: that's for general UI right? Not specific to UP5?

Comment (elfenheart) [reply cdap-ui/README.rst:42]: specific to applying User Pack

File: cdap-ui/README.rst (hunk 3 above)
Comment (bdmogal) [reply cdap-ui/README.rst:97]: when it renders, it should render correctly in rst. The line width is not fixed in the entire file currently

File: cdap-ui/README.rst (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/README.rst:42]: general UI is using the Node we ship. I think applying UP is using the system's Node

Comment (bdmogal) [reply cdap-ui/README.rst:42]: added a line

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Pull request merged**